src/components/WeeklyCalendar/week/helpers/getCurrentColor.js

```diff
@@ -0,0 +1,13 @@
+import { isValidColor } from '../../../../styles/helpers/color';
+
+export default function getCurrentColor(color1, color2, theme) {
```

Comment:
color1 and color2 are not meaningful names, in order to know what are they I need to go to where it is called and then see who is each one. Instead, we can name them as color and backgroundColor also if you use only one argument with an object we can name each parameter and then we can not rely on argument order but in argument names:
```
function getCurrentColor({ color, backgroundColor, theme }) {
```